src/fastoad/notebooks/02_explanation/04_OpenMDAO.ipynb

```diff
@@ -0,0 +1,1010 @@
+{
```

Comment:
<blockquote>... This enables to transform the minimization in a maximization.  </blockquote><p>I would suggest, for more accuracy :</p><p>"A common usage is to set the scaler to -1 to transform the minimization in a maximization."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
Changed in dfe233f7